docs/src/main/paradox/server.md

```diff
@@ -10,89 +10,94 @@ Then add the Akka gRPC plugin to your build:
 
 sbt
```

Comment:
Those tabs, and also the ones I added to gradle.md, are not correctly rendered by Paradox.
It's broken in master also.

We have similar in several places in Akka docs, e.g. in https://raw.githubusercontent.com/akka/akka/master/akka-docs/src/main/paradox/cluster-routing.md

Reply:
Hmm, I thought that was what you fixed with the indent. If it doesn't help anything, do we really want the sample code/config indented one step?

Reply:
the indentation was one (of many) attempts to make it render the tabs at all

Reply:
Markdown in anger!

Reply:
I worked around the rendering problem and created issue https://github.com/akka/akka-grpc/issues/294